fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -123,6 +123,10 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	double BG_REBALANCE_POLLING_INTERVAL;
 	double BG_REBALANCE_SWITCH_CHECK_INTERVAL;
 	double DD_QUEUE_LOGGING_INTERVAL;
+	double DD_QUEUE_COUNTER_REFRESH_INTERVAL;
+	double DD_QUEUE_COUNTER_MAX_LOG; // max trace event the counter will generate in a refresh round
```

Comment:
I feel like these could be a little more descriptive.
```suggestion
	double DD_QUEUE_COUNTER_MAX_LOG; // max number of servers for which trace events will be generated in each round of DD_QUEUE_COUNTER_REFRESH_INTERVAL duration
```